### PR TITLE
Add OASys import risk information screens to Apply from AP (Part 1)

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

This PR is part 1 of adding screens to our Apply journey to allow users to import information on the person to be placed from OASys. On the screen we add in this PR, a user can select which of the optional OASys sections they wish to import. These will be imported in the next PR

### Before
localhost_3000_applications_b826b0ca-8e28-4d80-ba14-f3765b68ced0 (1)
localhost_3000_applications_b826b0ca-8e28-4d80-ba14-f3765b68ced0_tasks_check-your-answers_pages_review (1)

### After
localhost_3000_applications_b826b0ca-8e28-4d80-ba14-f3765b68ced0_tasks_oasys-import_pages_optional-oasys-sections
localhost_3000_applications_b826b0ca-8e28-4d80-ba14-f3765b68ced0
localhost_3000_applications_b826b0ca-8e28-4d80-ba14-f3765b68ced0_tasks_check-your-answers_pages_review

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Have all changes to any dependencies been deployed to both `preprod` and
    `production` in a backwards compatible way? (This includes our API,
    infrastructure, third-party integrations and any secrets or environment variables)
- [ ] Has a required data migration been included in this change or been done in
    advance?

## Post merge checklist

Once we've merged we now need to release this through to production before
considering it done.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-temporary-accommodation-ui)
and work through the following steps:

- [ ] Has the product manager asked to provide approval for this feature?
  - [ ] Has the product manager approved?
- [ ] Manually approve release to preprod
- [ ] Verify change released to preprod
- [ ] Manually approve release to prod
- [ ] Verify change released to production

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Add documents screens to Apply from AP](https://trello.com/c/cgvK9O2u/860-add-documents-screens-to-apply-from-ap)

**Comment by bryangaledxw** [comment]
[Add OAsys import/risk information screens to Apply from AP](https://trello.com/c/1wg76KUW/861-add-oasys-import-risk-information-screens-to-apply-from-ap)

File: server/utils/formUtils.test.ts [hunk 1]
```diff
@@ -358,4 +363,66 @@ describe('formUtils', () => {
       ])
     })
   })
+
+  describe('validPostcodeArea', () => {
+    it('when passed a postcode area it returns true', () => {
+      expect(validPostcodeArea('HR1')).toBe(true)
+    })
+
+    it('when passed a lowecase postcode area it returns true', () => {
```
Comment (patrickjfl) [inline server/utils/formUtils.test.ts]: 'lowercase' - typo

**Review by patrickjfl — APPROVED** [review]
LGTM. Am I right in thinking the E2E testing for Apply has been postponed for now?

**Comment by bryangaledxw** [comment]
@patrickjfl Yeah - ideally it'd go in with this, but it's a non-trivial piece of API work and Apply is deprioritised for now

**Pull request merged**